cmd/cstorvolumeclaim/controller.go

```diff
@@ -156,13 +161,21 @@ func (c *CVCController) syncCVC(cvc *apis.CStorVolumeClaim) error {
 		// We choose to absorb the error here as the worker would requeue the
 		// resource otherwise. Instead, the next time the resource is updated
 		// the resource will be queued again.
-		runtime.HandleError(fmt.Errorf("cvc must be publish/attached to Node: %v", cvc))
+		runtime.HandleError(fmt.Errorf("cvc must be publish/attached to Node: %+v", cvc))
+		c.recorder.Event(cvc, corev1.EventTypeWarning,
+			Provisioning,
+			fmt.Sprintf(MessageCVCPublished, cvc.Name),
+		)
 		return nil
 	}
 
 	if cvc.Status.Phase == apis.CStorVolumeClaimPhasePending {
-		klog.Infof("provisioning cstor volume %+v", cvc)
-		cvc, err = c.createVolumeOperation(cvc)
+		klog.V(2).Infof("provisioning cstor volume %+v", cvc)
+		_, err = c.createVolumeOperation(cvc)
```

Comment:
On success case, we need to deal with updatedCVC object

Reply:
not required now.. as we are not using the cvc object returned from the volumeoperations call.